admin_manual/release_notes.rst

```diff
@@ -12,6 +13,61 @@ Release Notes
+
+- **Configurable solution for indistinguishable user display names**
+In the ownCloud sharing dialog users are displayed according to their display name. As users can choose their display name in self-service (can be disabled in config.php) and display names are not unique it is possible that a user can't distinguish sharing results. To cover this case the displayed user identifiers are now configurable. Administrators can configure to additionally display mail addresses or user ids. Please check if this new feature better suits your needs.
+
+- **Added "occ files:scan" repair mode to repair filecache inconsistencies**
+We recommend to use this command when directed to do so in the upgrade process (TODO more info)
+Please check the ownCloud documentation for more information. **LINK (https://github.com/owncloud/documentation/issues/3446)**
+
+- **Detailed mode for "occ security:routes"**
+Administrators can use the output of this command when using a network firewall to check the appropriateness of configured rules or to get assistance when setting up.
+
+- **Added mode of operations to differentiate between single-instance or clustered setup**
+As ownCloud needs to behave differently when operating in a clustered setup we added a config.php option ``'operation.mode' => 'single-instance' or 'clustered-instance'``. 
+Currently the Market App (ownCloud Marketplace integration) does not support clustered setups and can do harm when used for installing or updating apps. The new config setting prevents this and other actions that are undesired in cluster mode.
+**Therefore when operating in a clustered setup it is mandatory to set this option.**
+Please check the ownCloud documentation for more information. **LINK (https://github.com/owncloud/documentation/pull/3506)**
+
+- **Added occ dav:cleanup-chunks command to clean up expired uploads**
+When file uploads are interrupted by any reason already uploaded file parts (chunks) stay on the storage to be able to resume the file upload in a following upload run. Resuming an upload is only possible until the partial upload is expired and deleted, respectively. To cleanup chunks (expire and delete) originating from unfinished uploads administrators can use this newly introduced command. The expiry time default is two days but can be specified as a parameter of this command.
+**It is recommended to configure CRON to execute this background job regularly (it is not included in the regular ownCloud background jobs to give administrators more flexibility in scheduling)**. Please check the ownCloud documentation for more information. **LINK (https://github.com/owncloud/documentation/pull/3522)**
+
+- **Administrators can now exclude files from integrity check in config.php**
+When administrators did intentional changes to the ownCloud code they now have the ability to exclude certain files from the integrity checker.
```

Comment:
AKA how to shoot yourself in the foot ...

Reply:
now that you mention it I'm not 100% that it works that way.

the original purpose was to exclude non-OC files (often server specific hidden files) that people have lying around inside the OC folder.

need to test whether excluding already signed files from integrity check works.

Reply:
Agree to @PVince81 
would not state this as the reason for having that implemented which is different to this text.
(Do not open Pandoras Box...)
Please see the link to get an impression of: https://github.com/owncloud/core/pull/29126/files
Please reference to the documentation

Reply:
I have tested this with the 10.0.4RC1 tarball and can confirm that you can exclude patched files from integrity check as well as any new file that you might have created. So the description is good.